### PR TITLE
Added antibody sequence to naive B cells

### PR DESCRIPTION
**Pull request by fyffep**

### CODE REVIEW

File: PhysiCell/config/PhysiCell_settings.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
                     <model code="101" name="necrosis">
                         <death_rate units="1/min">0.0</death_rate>
                         <phase_transition_rates units="1/min">
-                            <rate start_index="0" end_index="1" fixed_duration="false">9e9</rate>
+                            <rate start_index="0" end_index="1" fixed_duration="false">9000000000.0</rate>
```
Comment (fyffep) [inline PhysiCell/config/PhysiCell_settings.xml:104]: This was changed automatically when I launched model builder, so I will keep it like this.

Comment (elmbeech) [reply PhysiCell/config/PhysiCell_settings.xml:104]: yes, i know, i think the studio does this. no worries!

**Comment by elmbeech** [comment]
Hi Pete!

This is cool! But I have a question. If I read it right, then you made the antibody and antigen sequence a sequence of doubles.
Would it not make more sense to have a sequence of chars?
Please correct me if I read the code wrong. I am rather new to C++.

Elmar

**Comment by fyffep** [comment]
Elmar,I only used double because I wanted to leverage built-in Physicell functions. Thanks for your review! PeteOn Mar 14, 2023, at 3:50 PM, elmar bucher ***@***.***> wrote:﻿
Hi Pete!
This is cool! But I have a question. If I read it right, then you made the antibody and antigen sequence a sequence of doubles.
Would it not make more sense to have a sequence of chars?
Please correct me if I read the code wrong. I am rather new to C++.
Elmar

**Comment by elmbeech** [comment]
This is totally fine!  As I said, I am very new to C++ and just getting used to PhysiCell myself. 
And C++ is a bit lower level than Python3. 
I figured out double is actually quite nice. It can handle real, integer, and char. 
I just have to translate to the ASCII. 
The code you wrote works very nice! 
One thing I like to change: the variable name foreignAntigen. 
Antigens are always kind of "foreign" .   Keeping in the logic of antibodySequence (which is always own) could not simply name it antigenSequence?  
That's all! 
I happily merge and if you are ok, do the change later.

**Comment by elmbeech** [comment]
Cool code! Thanks a bunch.

**Comment by fyffep** [comment]
I wouldn’t worry too much about the names. If we do, we might never get our work done. I’m glad that the code is a good start for you. PeteOn Mar 14, 2023, at 8:01 PM, elmar bucher ***@***.***> wrote:﻿
Cool code! Thanks a bunch.

**Comment by elmbeech** [comment]
Cool, if you don't worry about the name, then I change it because I do. Thank you!  
I think we will get our work done. 
Will now focus on the aligning and measuring the distance of the sequences.

**Pull request merged**